### PR TITLE
[issue#198] No more GUI option for "additional transaction fee" :)

### PR DESCRIPTION
**Pull request by belovachap**
:wave: @peerchemist @hrobeers @backpacker69 

This fella https://github.com/peercoin/peercoin/issues/198

I wrote a thing! Pulled back the changes to just remove text and option from the GUI options. Also ran the translation string extraction script and whatever `lupdate` is (some sort of QT tool?).

Can still send transactions on the testnet... so that's encouraging :rainbow:

### CODE REVIEW

File: src/bitcoinrpc.cpp [hunk 1]
```diff
@@ -448,7 +448,6 @@ static const CRPCCommand vRPCCommands[] =
     { "verifymessage",          &verifymessage,          false,     false },
     { "getwork",                &getwork,                true,      false },
     { "listaccounts",           &listaccounts,           false,     false },
-    { "settxfee",               &settxfee,               false,     false },
```
Comment (backpacker69) [inline src/bitcoinrpc.cpp]: there's really no need to remove that rpc call, just remove it from gui part if you must.

**Comment by belovachap** [comment]
will stick to just the gui options :+1:

**Comment by hrobeers** [comment]
@belovachap better rewrite history of this pull request.
now we have a commit removing lines and add them back by a later one.

So just do a `git reset --soft <txid>` and commit all in one commit and do a `git push -f`

This will also make it easier to cherry pick in later code rebases.

**Comment by hrobeers** [comment]
@backpacker69 @belovachap 
This looks good to me.
Haven't done any testing yet, I leave that up to you guys since it doesn't touch the protocol.

File: src/qt/bitcoinstrings.cpp [hunk 2]
```diff
@@ -17,7 +17,8 @@ QT_TRANSLATE_NOOP("bitcoin-core", ""
 "If the file does not exist, create it with owner-readable-only file "
 "permissions.\n"
 "It is also recommended to set alertnotify so you are notified of problems;\n"
-"for example: alertnotify=echo %%s | mail -s \"Peercoin Alert\" admin@foo.com\n"),
+"for example: alertnotify=echo %%s | mail -s \"Peercoin Alert\" admin@foo."
+"com\n"),
```
Comment (sandakersmann) [inline src/qt/bitcoinstrings.cpp]: Is this correct?

Comment (hrobeers) [reply src/qt/bitcoinstrings.cpp]: It possibly is an accidental return.
But it doesn't impact compilation, it's just syntactically different.
Good catch though, will revert this later.

**Pull request merged**